source/MaterialXGenShader/Nodes/SourceCodeNode.cpp

```diff
@@ -34,7 +34,7 @@ void SourceCodeNode::initialize(const InterfaceElement& element, GenContext& con
     if (_functionSource.empty())
     {
         FilePath file(impl.getAttribute("file"));
-        file = context.resolveSourceFile(FilePath("libraries") / file);
```

Comment:
No need for putting "libraries" here, if the search path on the context is set correctly. This forces all apps to have a "libraries" root folder which is not always the case.